operations/patch.go

```diff
@@ -93,12 +98,10 @@ func Patch() cli.Command {
 				return errors.Wrap(err, "can't test for uncommitted changes")
 			}
 
-			useMbox := true
 			if uncommittedChanges {
-				if params.Uncommitted || conf.UncommittedChanges {
-					grip.Info("Patches with uncommitted changes cannot be enqueued on a commit queue.")
-					useMbox = false
-				} else {
+				if params.PreserveCommits {
+					grip.Infof("Uncommitted changes are omitted from patches when commits are preserved")
```

Comment:
ended with punctuation.
heretofore we've warned a user when there are uncommitted changes that aren't going to be included in their patch. I think the expectation has been that users will assume the contents of their patch will match their working directory.

Reply:
True, I actually think now that this should be more strict if they are setting this flag, because you'd think they'd intended to merge their working directory. Let's continue conversation in the other thread 👍